components/UI/Map/AhbapClusterGroup.tsx

```diff
@@ -78,11 +78,18 @@ export const AhbapClusterGroup = ({ data }: Props) => {
             })}
             eventHandlers={{
               click: (e) => {
+                let description = cluster.properties.description;
```

Comment:
You have already set the "description" value to a variable. You can do comparison operations on this variable.

Reply:
not sure what you mean here,  but I mutate the variable I created, if `cluster.properties.description` is an object, then description variable should its value, otherwise it can stay as description which should be a string.